validator/sawtooth_validator/stats/__init__.py

```diff
@@ -0,0 +1,7 @@
+def OpenTSDBReporter(*args, **kwargs):
```

Comment:
Is this also taken from the pyformance library?  If so, I would put the appropriate comments with license and copyright notices in the file.

Otherwise, it needs the Intel copyright header